src/Features/Core/Portable/EmbeddedLanguages/StackFrame/StackFrameCompilationUnit.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.EmbeddedLanguages.Common;
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.StackFrame
+{
+    internal class StackFrameCompilationUnit
+    {
+        public readonly EmbeddedSyntaxTrivia<StackFrameKind>? AtTrivia;
+        public readonly StackFrameMethodDeclarationNode MethodDeclaration;
+        public readonly EmbeddedSyntaxTrivia<StackFrameKind>? InTrivia;
```

Comment:
in general, nodes never point at trivia.  it's something attached to a token... i feel semi strongly we should preserve that concept.

if you do want to have this hear, i would make this an optional token then.

Reply:
If I make this trivia attached to a token, is there a concept of a beginning of line token (like EOL)? I'd imagine that's where we want `at`. The `in` could be trivia on the token representing the file path (once I implement that) and not be required as top level of the compilation unit.

Reply:
Removed `AtTrivia`, will remove `InTrivia` next.